akka-actor-typed-tests/src/test/java/jdocs/akka/typed/InteractionPatternsTest.java

```diff
@@ -12,6 +12,8 @@
 import akka.actor.typed.javadsl.Behaviors;
 import akka.actor.typed.javadsl.TimerScheduler;
 import akka.testkit.typed.scaladsl.TestProbe;
+import akka.util.Timeout;
+import javafx.scene.control.DateCell;
```

Comment:
Magic IntelliJ import?

Reply:
God damnit IntelliJ, I told you to stop doing that...

Reply:
Exclude javafx ;) I don't think you'll be using it much in Akka... :)

Reply:
ha, ha, intellij know your are a javafx programmer, you can't hide it any more

Reply:
Watch out when it'll start including actionscript in my PRs... 😉